jwst/datamodels/model_base.py

```diff
@@ -306,14 +313,31 @@ def close(self):
             if fd is not None:
                 fd.close()
 
-    def get_envar(self, name, value):
+    def get_envar_as_boolean(self, name, default=False):
```

Comment:
This should probably be a private method `_get_envar_as_boolean`.  I don't think we want it as part of the pubic API of the class.